Add style values for smaller device on job listing page

miyabitanimchi opened the pull request:
Tailored styles for smaller device on `/joblisting/:id` page. Please check when you can, thank you!

BlackStone8960 reviewed (commented):
It looks good! 
But can you just fix following styles?

1. The Skill Tags are overlapped if they consists of multiple lines
2. The margin between left part and right part in summary lists is narrower than mockup

<img width="331" alt="Screen Shot 2021-12-30 at 2 54 16 PM" src="https://user-images.githubusercontent.com/60183777/147793094-f6e31eb0-5a9f-40ac-9153-a9fa9285d81d.png">

miyabitanimchi commented:
@BlackStone8960 Thank you for reviewing,  I fixed as follows:

1. The overlap of skill tags as you mentioned.
2. The width between the left part and the right part as you mentioned (smaller device)
3. I noticed the style was off because of the replacement of button component, so I added the style again.
4. Added margin under the 続きを見る "read more" button.

Please check again, thank you!
